naiserator-dev.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: emottak-monitor
   namespace: default
```

Comment:
Det kan endres fra default til team namespace, men da må man også slette den fra det namespacet etterpå (https://doc.nais.io/clusters/team-namespaces/#on-prem-migration-to-team-namespaces)

`namespace: team-emottak`